README.md

```diff
@@ -35,14 +35,36 @@ npm run ui
 
 4. Open a new Terminal tab and run the web server
 
+If running on MacOS
 ```
 npm run server
 ```
 
+If running on Windows
+```
+npm run server-windows
+```
+
 5. UI should open in the browser at `localhost:6002`
 
 If using VSCode, be sure to disable the Prettier: Format on Save feature to prevent unwanted changes to the code styles.
 
+## Using a local database 
+
+If you want to set a local databe follow this steps:
+
+1. Download MySQL server (Usually it comes with others features).
```

Comment:
Maybe provide a link here to help the reader find the correct version of MySQL to download?